.github/workflows/ci.yml

```diff
@@ -0,0 +1,113 @@
+name: CI
+on: [push, pull_request]
+env:
+  HUGO_VERSION: 0.26
+  SCALA_VERSION: 2.12.10
```

Comment:
So this `SCALA_VERSION` is for the jobs other than `validate`, which has its defined as `SCALA_VERSION: ${{ matrix.scala }}` from the matrix?

Reply:
Yes. And I ended up hardcoding a similar Java 8 elsewhere.